scripts/mainnet_market_setup_data.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "marketName": "USDC.WBTC.20201214.C.20000",
+    "priceRatio": "200000000000000000000",
+    "expirationDate": 1607904000,
+    "marketStyle": 1,
+    "exerciseFeeBasisPoints": 0,
+    "closeFeeBasisPoints": 0,
+    "claimFeeBasisPoints": 0
+  },
+  {
+    "marketName": "USDC.WBTC.20201214.C.10000",
+    "priceRatio": "100000000000000000000",
+    "expirationDate": 1607904000,
+    "marketStyle": 1,
+    "exerciseFeeBasisPoints": 0,
+    "closeFeeBasisPoints": 0,
+    "claimFeeBasisPoints": 0
+  },
+  {
+    "marketName": "USDC.WBTC.20201214.P.10000",
+    "priceRatio": "5000000000000000",
+    "expirationDate": 1609372800,
+    "marketStyle": 1,
+    "exerciseFeeBasisPoints": 0,
+    "closeFeeBasisPoints": 0,
+    "claimFeeBasisPoints": 0
+  },
+  {
+    "marketName": "USDC.WBTC.20201214.P.20000",
```

Comment:
Same here. Deep ITM put seems not too desirable

Reply:
fixed, see PR comment